thirdparty/Eluant.dll.config.in

```diff
@@ -1,3 +1,4 @@
 <configuration>
   <dllmap os="linux" dll="lua51.dll" target="@LIBLUA51@" />
+  <dllmap os="openbsd" dll="lua51.dll" target="liblua5.1.so" />
```

Comment:
This probably wants to be using `@LIBLUA51@` for compatibility with the lua-detection script.

Reply:
in that case I'll have to add some logic for filling in the correct `@LIBLUA51@` in those scripts. Slightly more complex, will work on it later.

Reply:
We already have logic to do this (`thirdparty/configure-native-deps.sh`), and it must be working already - the config file isn't parsed at all if that fails.